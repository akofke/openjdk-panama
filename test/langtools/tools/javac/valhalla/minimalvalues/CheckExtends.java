/*
 * @test /nodynamiccopyright/
 * @summary Values may not extend
 * @modules jdk.incubator.mvt
 * @compile/fail/ref=CheckExtends.out --should-stop:ifError=PARSE -XDrawDiagnostics -Werror  -Xlint:values CheckExtends.java
 */
@jdk.incubator.mvt.ValueCapableClass
final class CheckExtends extends Object {
}
