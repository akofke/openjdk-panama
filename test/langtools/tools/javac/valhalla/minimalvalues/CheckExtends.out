- compiler.warn.incubating.modules: jdk.incubator.mvt
- compiler.err.warnings.and.werror
CheckExtends.java:8:7: compiler.warn.value.may.not.extend
1 error
2 warnings
